### PR TITLE
Create REVIEWME.txt

### PR DESCRIPTION
**Pull request by MichaelWong1024**

### CODE REVIEW

File: REVIEWME.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Please review me!
```
Comment (ncash6) [inline REVIEWME.txt:1]: I found your visualizations interesting since the two rust spectrograms appeared to be more spread across the frequency band than the python implementations. It would be helpful to see the audio files plotted in the time domain to observe the loudness spread in comparison between the python and rust implementations.

Your functions for LFO and Vibrato were all very clear in their purpose, and the tests had enough comments to determine their purpose. Towards the end of the file is a Modulator and Vibrato struct, respectively, and I am unclear as the purpose of it. It would be useful to include a short description of all structs utilized at the top of the rust script.

The main file has two instances of mod lfo;. I am unsure if this duplication was intentional.

Your linear interpolation test in ring_buffer appeared thorough in considering edge cases.

Great work!
